Week-2/Animation/styles.css

```diff
@@ -0,0 +1,78 @@
+div#main {
+    position: relative;
+    height: 200px;
+    width: 200px;
+    margin-top: 100px;
+    margin-left: 700px;
+  }
+  div#main .content {
+    position: absolute;
+    text-align: center;
+    width: 200px;
+    top: 40%;
+    font-size: 24pt;
+  }
+  
+  div#main .topleft {
+    position: absolute;
+    width: 100px;
+    height: 100px;
+    top: 0px;
+    left: 0px;
+    background: red;
```

Comment:
Please use css `variables` for declaring colors.

Reply:
Used CSS Variables for the background color.